tests/src/test/scala/whisk/core/cli/test/WskWebActionsTests.scala

```diff
@@ -250,7 +172,7 @@ trait WskWebActionsTests
                 response.statusCode shouldBe 200
                 response.header("Access-Control-Allow-Origin") shouldBe "*"
                 response.header("Access-Control-Allow-Methods") shouldBe "OPTIONS, GET, DELETE, POST, PUT, HEAD, PATCH"
-                response.header("Access-Control-Allow-Headers") shouldBe "Authorization, Content-Type"
+                response.header("Access-Control-Allow-Headers") shouldBe "Authorization, Content-Type, User-Agent"
```

Comment:
@dubeejw - understood.  just wanted to be clear about the default behavior.  the docs currently do not tell the user that the Access-Control-Allow-Headers preflight response header is added
https://github.com/apache/incubator-openwhisk/blob/master/docs/webactions.md#options-requests